frontend/src/components/profile/components/profile-info/profile-info.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { IUser } from 'common/interfaces/user';
+import { getDateFormat } from 'helpers';
+import clsx from "clsx";
+import styles from './styles.module.scss';
+
+type Props = {
+  user: IUser | null
+  edit: () => void
+}
```

Comment:
```suggestion
type Props = {
  user: IUser;
  edit: () => void
}
```
use on<Element><Event> for component props cb